packages/schema/src/types/nuxt.ts

```diff
@@ -4,6 +4,57 @@ import type { NuxtHooks, NuxtLayout, NuxtMiddleware } from './hooks'
 import type { Component } from './components'
 import type { NuxtOptions } from './config'
 
+export interface NuxtPlugin {
+    /** @deprecated use mode */
+    ssr?: boolean
+    src: string
+    mode?: 'all' | 'server' | 'client'
+    /**
+     * This allows more granular control over plugin order and should only be used by advanced users.
+     * Lower numbers run first, and user plugins default to `0`.
+     *
+     * Default Nuxt priorities can be seen at [here](https://github.com/nuxt/nuxt/blob/9904849bc87c53dfbd3ea3528140a5684c63c8d8/packages/nuxt/src/core/plugins/plugin-metadata.ts#L15-L34).
+     */
+    order?: number
```

Comment:
```suggestion
  /** @deprecated use mode */
  ssr?: boolean
  src: string
  mode?: 'all' | 'server' | 'client'
  /**
   * This allows more granular control over plugin order and should only be used by advanced users.
   * Lower numbers run first, and user plugins default to `0`.
   *
   * Default Nuxt priorities can be seen at [here](https://github.com/nuxt/nuxt/blob/9904849bc87c53dfbd3ea3528140a5684c63c8d8/packages/nuxt/src/core/plugins/plugin-metadata.ts#L15-L34).
   */
  order?: number
```

Reply:
Wondering the same, as I tried changing quotes to double quotes, and it correctly detected the issue. Looking at the `ssr`'s indent, for some reason it seems to be fine...

![image](https://github.com/nuxt/nuxt/assets/48835293/979ba1f3-be76-4675-b46a-c57f91eeb530)

Reply:
It appears to have something to do with the folder name `types` this file is in.